cmd/manager/main.go

```diff
@@ -6,32 +6,24 @@ import (
 	"flag"
 	"log"
 	"net"
-	"strings"
+	"time"
 
 	pb "github.com/kubeflow/katib/pkg/api"
 	kdb "github.com/kubeflow/katib/pkg/db"
 	"github.com/kubeflow/katib/pkg/manager/modelstore"
-	tbif "github.com/kubeflow/katib/pkg/manager/visualise/tensorboard"
-	"github.com/kubeflow/katib/pkg/manager/worker"
-	k8swif "github.com/kubeflow/katib/pkg/manager/worker/kubernetes"
 
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/reflection"
 )
 
 const (
-	namespace                = "katib"
-	port                     = "0.0.0.0:6789"
-	defaultEarlyStopInterval = 60
-	defaultSaveInterval      = 30
+	port = "0.0.0.0:6789"
 )
 
-var workerType = flag.String("w", "kubernetes", "Worker Type")
-var ingressHost = flag.String("i", "kube-cluster.example.net", "Ingress host for TensorBoard visualize")
+var ingressHost = flag.String("i", "kube-cluster.example.net", "Ingress host")
```

Comment:
What is this ingress used for?

Reply:
It was used when the ingress was used in example demo. 
We can delete it now.